src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultModelMetadata.cs

```diff
@@ -356,5 +356,14 @@ public override string TemplateHint
                 return DisplayMetadata.TemplateHint;
             }
         }
+
+        /// <inheritdoc />
+        public override IReadOnlyList<object> ValidatorMetadata
+        {
+            get
+            {
+                return new ReadOnlyCollection<object>(ValidationMetadata.ValiatorMetadata);
```

Comment:
Yeah I meant to clean this up but I forgot - wil fix. BTW `ReadOnlyCollection` is a wrapper, it doesn't do any copying.